virtcontainers/sandbox.go

```diff
@@ -1164,6 +1164,8 @@ func (s *Sandbox) CreateContainer(contConfig ContainerConfig) (VCContainer, erro
 			if len(s.config.Containers) > 0 {
 				// delete container config
 				s.config.Containers = s.config.Containers[:len(s.config.Containers)-1]
+				// need to flush change to persist storage
+				_ = s.storeSandbox()
```

Comment:
please don't ignore the error, at least print it out

Reply:
Ok